pkg/network/tracer/offsetguess/tracer.go

```diff
@@ -273,7 +275,7 @@ func GetIPv6LinkLocalAddress() (*net.UDPAddr, error) {
 			continue
 		}
 		for _, a := range addrs {
-			if strings.HasPrefix(a.String(), "fe80::") {
+			if strings.HasPrefix(a.String(), "fe80::") && !strings.HasPrefix(i.Name, "dummy") {
```

Comment:
this was breaking tests?

Reply:
Yes, on Amazon Linux. It was choosing the dummy interface added for DNAT, rather than `eth0` (or equivalent)

Reply:
`error guessing offsets: listen udp6 [fe80::845c:1aff:fe71:74bf%dummy1]:0: bind: no such device`

Reply:
ah I see, thanks